docs/services/slam/cartographer/_index.md

```diff
@@ -12,9 +12,11 @@ aliases:
 
 [The Cartographer Project](https://github.com/cartographer-project) performs dense SLAM using LIDAR data.
 
+To easily integrate Cartographer into the Viam ecosystem, Viam provides [`viam-cartographer`](https://github.com/viamrobotics/viam-cartographer), a [modular resource](/program/extend/modular-resources/) which wraps Cartographer. This resource provides the `cartographer-module` binary for use directly on your SLAM-enabled robot.
```

Comment:
```suggestion
To easily integrate Cartographer into the Viam ecosystem, Viam provides [`viam-cartographer`](https://github.com/viamrobotics/viam-cartographer), a module which provides Cartographer as a
[modular resource](/program/extend/modular-resources/).
```
Suggesting as I think something like this would help users to understand the modular resource part, it's somewhat more in line with our other phrasing around modules / modular resources? the modules part has to be changed but rest is opt